contracts/vaults/BaseVaults/base/RibbonVault.sol

```diff
@@ -401,7 +401,7 @@ contract RibbonVault is
      * @notice Initiates a withdrawal that can be processed once the round completes
      * @param numShares is the number of shares to withdraw
      */
-    function initiateWithdraw(uint256 numShares) external nonReentrant {
+    function _initiateWithdraw(uint256 numShares) internal {
```

Comment:
I dunno man, this is dangerous we remove `nonReentrant` on the most important method.  If the deriving vault contract forgets to mark `nonReentrant`, could be a bad time.

Reply:
Looks cleaner to put the modifier on the top level function rather than an internal one